libs/generic-view/feature/src/lib/use-dev-views/views/contacts-view.ts

```diff
@@ -16,7 +16,70 @@ const view: View = {
         columns: ["1fr"],
       },
     },
-    childrenKeys: ["importContactsButton", "contactsLoader"],
+    childrenKeys: ["contactsPanel", "contactsLoader"],
+  },
+  contactsPanel: {
+    component: "block-box",
+    childrenKeys: ["contactsSearch", "contactsButtonActions"],
+    layout: {
+      gridLayout: {
+        rows: [],
+        columns: ["280px", "344px"],
+        justifyContent: "space-between",
+      },
+    },
+  },
+  contactsSearch: {
+    component: "form",
+    config: {
+      formOptions: {
+        defaultValues: {
+          searchedContact: undefined,
+        },
+      },
+    },
+    childrenKeys: ["contactsSearchWrapper"],
+  },
+  contactsSearchWrapper: {
+    component: "block-plain",
+    childrenKeys: ["contactsSearchInput"],
+  },
+  contactsSearchInput: {
+    component: "form.searchInput",
+    config: {
+      label: "Search all contacts",
+      name: "searchedContact",
+    },
+  },
+  contactsButtonActions: {
+    component: "modal.buttons",
```

Comment:
Better to use `block-plain` with proper layout (flexLayout with direction and gap) or (gridLayout with columns [1fr,1fr] and gap)